.circleci/config/commands/@deploy.yml

```diff
@@ -19,5 +19,5 @@ deploy_env:
           export DEPLOY_DOWNTIME=<< parameters.downtime >>
           export PACKAGE=<< parameters.package_path >>
           export DEPLOY_DB_VERSION=$(cat deployment/DB_VERSION)
-
-          cd /infrastructure && make deploy
+          
+          cd /infrastructure && make vault-config-update-${DEPLOY_ENV} deploy
```

Comment:
We should perhaps consider having a PHONY `vault-config-update` target in future

Reply:
https://github.com/orgs/aeternity/projects/3#card-78170842